### PR TITLE
style: add delay to BulkActionsToolbar

### PR DESCRIPTION
**Pull request by ahalyha**
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. Use imperative, present tense in your commit description ("change", not "changed" or "changes") without uppercases or period (.) at the end.

# Checklist
- [ ] The implementation has been manually tested and complies with Textkernel [browser support guidelines](https://textkernel.com/browser-support/)
- [ ] The implementation complies with [accessibility](CONTIRBUTING.md#accessibility) standards.
- [ ] The component has a [displayName](CONTRIBUTING.md#display-names) defined.
- [ ] The component comes with a detailed [PropTypes](CONTRIBUTING.md#component-props) (and defaultProps) definition.
- [ ] Component PropTypes are sufficiently described / documented.
- [ ] There is [a story](CONTRIBUTING.md#component-showcases) in Storybook.

### CODE REVIEW

File: src/components/BulkActionsToolbar/BulkActionsToolbar.tsx [hunk 1]
```diff
@@ -25,6 +25,7 @@ export type BulkActionsToolbarAction = {
     label: string;
     icon?: React.ReactElement;
     tooltip?: string;
+    delay?: number;
```
Comment (mukiienko) [inline src/components/BulkActionsToolbar/BulkActionsToolbar.tsx:28]: What do you think about renaming the prop to `tooltipDelay`?

File: src/components/BulkActionsToolbar/BulkActionsToolbar.tsx [hunk 2]
```diff
@@ -73,7 +74,7 @@ export const BulkActionsToolbar: React.FC<Props> = (props) => {
                     // be able to show the tooltip on disabled button.
                     // https://github.com/atomiks/tippyjs-react/issues/123
                     return (
-                        <Tooltip key={action.label} content={action.tooltip}>
+                        <Tooltip key={action.label} content={action.tooltip} delay={action.delay}>
```
Comment (mukiienko) [inline src/components/BulkActionsToolbar/BulkActionsToolbar.tsx:77]: `delay` is optional and I'm wondering how `Tooltip` component will behave if the value will be `undefined`.

Comment (mukiienko) [inline src/components/BulkActionsToolbar/BulkActionsToolbar.tsx:77]: Do we need to add delay for other Tooltips in this component?

File: src/components/BulkActionsToolbar/BulkActionsToolbar.tsx (hunk 1 above)
Comment (ahalyha) [reply src/components/BulkActionsToolbar/BulkActionsToolbar.tsx]: yes

File: src/components/BulkActionsToolbar/BulkActionsToolbar.tsx (hunk 2 above)
Comment (ahalyha) [reply src/components/BulkActionsToolbar/BulkActionsToolbar.tsx]: I am not sure, but visually it works fine

Comment (ahalyha) [reply src/components/BulkActionsToolbar/BulkActionsToolbar.tsx]: yes

**Review by mukiienko — APPROVED** [review]

**Pull request merged**